ui/src/data_explorer/containers/DataExplorer.js

```diff
@@ -6,15 +6,17 @@ import _ from 'lodash'
 
 import QueryMaker from '../components/QueryMaker'
 import Visualization from '../components/Visualization'
+import WriteDataForm from '../components/WriteDataForm'
 import Header from '../containers/Header'
-import ResizeContainer from 'src/shared/components/ResizeContainer'
+import ResizeContainer from 'shared/components/ResizeContainer'
```

Comment:
nice catch. though if you do this, you might change the other relative paths above to absolute as well. that being said, this change is technically out of scope for this PR, i'd either recommend all or none.